### PR TITLE
feat: passwordless strategy

### PR DESCRIPTION
**Pull request by hperl**
## Related Issue or Design Document

Part of https://github.com/ory-corp/cloud/issues/5470

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: src/react-components/ory/user-settings-screen.tsx [hunk 1]
```diff
@@ -153,12 +164,13 @@ const body = ({
         "oidc",
         "lookupSecret",
```
Comment (aeneasr) [inline src/react-components/ory/user-settings-screen.tsx:165]: Shouldn't this be `lookup_secret`?

File: src/react-components/ory/helpers/node.tsx [hunk 2]
```diff
@@ -106,37 +106,37 @@ export const uiTextToFormattedMessage = (
...
       /*
-      *
-      {
-      "text": {
-          "id": 1050015,
-          "text": "3r9noma8, tv14n5tu, ...",
-          "type": "info",
-          "context": {
-              "secrets": [
+                  *
```
Comment (aeneasr) [inline src/react-components/ory/helpers/node.tsx:109]: Revert?

**Review by aeneasr — COMMENTED** [review]
Do you know why look up secrets got removed in the snapshot?

See https://www.chromatic.com/pullrequest?appId=63b58e306cfd32348fa48d50&number=170&activeElementId=comparison-test-key-64e4a6953a35b8c89ddc9697-1200px&view=changes (scroll to bottom)

Screenshot 2023-12-18 at 11 07 47

File: src/react-components/ory/user-settings-screen.tsx (hunk 1 above)
Comment (aeneasr) [reply src/react-components/ory/user-settings-screen.tsx:165]: Looks correct.

File: src/react-components/ory/helpers/node.tsx (hunk 2 above)
Comment (jonas-jonas) [reply src/react-components/ory/helpers/node.tsx]: Done.

File: src/react-components/ory/user-settings-screen.tsx (hunk 1 above)
Comment (hperl) [reply src/react-components/ory/user-settings-screen.tsx:165]: Yes, it needs to be `lookup_secret`. I fixed it and it also resolved the UI diff.

**Pull request merged**